docs/pages/installation.mdx

```diff
@@ -48,56 +49,345 @@ supports most features on Windows 10 and later.*
+- Using the example Terraform module for deploying agents in [Deploy Agents with
+  Terraform](docs/pages/agents/deploy-agents-terraform.mdx).
+- Running the [one-line installation script](#one-line-installation-script) on
+  each Linux server where you want to install a Teleport agent.
 
-<Admonition type="note">
-`apt`, `yum`, and `zypper` repos don't expose packages for all distribution variants.
-When following installation instructions, you might need to replace `ID` with
-`ID_LIKE` to install packages of the closest supported distribution.
+If you are running a self-hosted Teleport Enterprise cluster, read our guidance
+in [Deploying a High-Availability Teleport
+Cluster](./deploy-a-cluster/introduction.mdx), which provides full requirements
+and Terraform modules for deploying Teleport Enterprise on your infrastructure.
 
-Currently supported distributions (and `ID`) are:
-- RHEL >= 7 (`rhel`)
-- CentOS >= 7 (`centos`)
-- Debian >= 9 (`debian`)
-- Ubuntu >= 16.04 (`ubuntu`)
-- Amazon Linux 2 and 2023 (`amzn`)
-- SLES >= 12 SP5, >= 15 SP5 (`sles`)
+### One-line installation script
 
-</Admonition>
+You can run a one-line command to install Teleport binaries on a Linux server.
+The command takes the Teleport version and edition to install, then uses
+information about the operating system where it runs to choose a package manager
+and install Teleport.
 
+First, assign environment variables based on your edition:
 
-### Installation instructions
+<Tabs>
+<TabItem label="Teleport Community Edition">
 
-(!docs/pages/includes/permission-warning.mdx!)
+```code
+$ TELEPORT_EDITION="oss"
+$ TELEPORT_VERSION="(=teleport.version=)"
+```
 
-(!docs/pages/includes/install-linux.mdx!)
+</TabItem>
+<TabItem label="Teleport Cloud">
 
-<Tabs>
-<TabItem scope="oss" label="Teleport Community Edition">
+The following commands show you how to determine the Teleport version to install
+by querying your Teleport Cloud account. This way, the Teleport installation has
+the same major version as the service that manages automatic updates:
 
-Check the [Downloads](https://goteleport.com/download/) page for the most
-up-to-date information.
+```code
+$ TELEPORT_EDITION="cloud"
+$ TELEPORT_DOMAIN=<Var name="example.teleport.sh" />
+$ TELEPORT_VERSION="$(curl https://$TELEPORT_DOMAIN/v1/webapi/automaticupgrades/channel/default/version | sed 's/v//')
+```
 
 </TabItem>
-<TabItem scope="cloud" label="Teleport Enterprise Cloud">
+<TabItem label="Teleport Enterprise">
 
-Check the [Cloud Downloads](./choose-an-edition/teleport-cloud/downloads.mdx) page for the most up-to-date
-information on obtaining Teleport binaries compatible with Teleport Cloud.
+```code
+$ TELEPORT_EDITION="enterprise"
+$ TELEPORT_VERSION="(=teleport.version=)"
+```
 
 </TabItem>
-
 </Tabs>
 
+Download and run the installation script on the server where you want to install
+Teleport:
+
+```code
+$ curl https://goteleport.com/static/install.sh | bash -s ${TELEPORT_VERSION?} ${TELEPORT_EDITION?} 
```

Comment:
Does this script install auto-updater package when installing a cloud edition?

Reply:
It does as of gravitational/next#2407